lca-main.yaml

```diff
@@ -563,6 +571,34 @@ Parameters:
       from the Recordings bucket."
     Default: 90
 
+  ExcludeVerboseContentValue:
```

Comment:
I prefer not to expose thse AppSync logging parameters in the main template, as they will clutter up the parameter list further, and will require README documentation which i don't see in this PR. Let's include it only in the nested AISTACK template, and you can always update just the nested stack to change them from the defaults.